packages/admin-next/dashboard/src/routes/products/product-create/utils.ts

```diff
@@ -70,19 +79,19 @@ export const decorateVariantsWithDefaultValues = (
     ...variant,
     title: variant.title || "",
     sku: variant.sku || "",
-    ean: variant.ean || "",
-    upc: variant.upc || "",
-    barcode: variant.barcode || "",
+    // ean: variant.ean || "",
+    // upc: variant.upc || "",
+    // barcode: variant.barcode || "",
     manage_inventory: variant.manage_inventory || false,
     allow_backorder: variant.allow_backorder || false,
     inventory_kit: variant.inventory_kit || false,
-    mid_code: variant.mid_code || "",
-    hs_code: variant.hs_code || "",
-    width: variant.width || "",
-    height: variant.height || "",
-    weight: variant.weight || "",
-    length: variant.length || "",
-    material: variant.material || "",
-    origin_country: variant.origin_country || "",
+    // mid_code: variant.mid_code || "",
+    // hs_code: variant.hs_code || "",
+    // width: variant.width || "",
+    // height: variant.height || "",
+    // weight: variant.weight || "",
+    // length: variant.length || "",
+    // material: variant.material || "",
+    // origin_country: variant.origin_country || "",
```

Comment:
question: should we not just delete all of these now?

Reply:
yes, I was just waiting for confirmation on linear but I'll remove this